### PR TITLE
Sponsorship Page 

### PR DESCRIPTION
**Pull request by dhruvilp**
Ready to Merge?

### CODE REVIEW

File: frontend/public/index.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
     <meta property="og:image" content="./assets/fb-banner-upper.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="Sign up for HackRU Spring 2019, taking place from March 9th to 10th.">
-    <link rel="shortcut icon" href="%PUBLIC_URL%/assets/fav.png">
```
Comment (thekabistro) [inline frontend/public/index.html:25]: Is there a reason why you removed `%PUBLIC_URL%`? We'll need this when we push to live I believe.

File: frontend/src/MLHBadge.jsx [hunk 2]
```diff
@@ -15,7 +15,7 @@ import React, { Component } from "react";
 class MLHBadge extends Component {
     render() {
         return (
-            <a id="mlh-trust-badge" style={{ display: "block", maxWidth: "100px", minWidth: "60px", position: "absolute", right: "50px", top: "0", width: "10%", zIndex: "10" }} href="https://mlh.io/seasons/na-2019/events?utm_source=na-hackathon&utm_medium=TrustBadge&utm_campaign=2019-season&utm_content=black" target="_blank" rel="noopener noreferrer">
+            <a id="mlh-trust-badge" style={{ display: "block", visibility: "visible", maxWidth: "100px", minWidth: "60px", position: "absolute", right: "50px", top: "0", width: "10%", zIndex: "10" }} href="https://mlh.io/seasons/na-2019/events?utm_source=na-hackathon&utm_medium=TrustBadge&utm_campaign=2019-season&utm_content=black" target="_blank" rel="noopener noreferrer">
```
Comment (thekabistro) [inline frontend/src/MLHBadge.jsx]: Is this needed?

File: readme.md [hunk 3]
```diff
@@ -1,53 +1 @@
...
--  [Overview of Style Guides + Some Examples](https://codeburst.io/5-javascript-style-guides-including-airbnb-github-google-88cbc6b2b7aa)
--  [Using ESLint](https://eslint.org/docs/user-guide/getting-started)
-
-### Testing and CI:
--  [Continuous Integration Essentials](https://codeship.com/continuous-integration-essentials)
--  [Getting Started with Jest](https://jestjs.io/docs/en/getting-started)
-
-### Version Control on Git:
--  [Git Cheat Sheet](https://www.atlassian.com/git/tutorials/atlassian-git-cheatsheet)
-
```
Comment (thekabistro) [inline readme.md:53]: Any particular reason to remove all this?  I figure it might be useful for someone down the line.

**Review by thekabistro — CHANGES_REQUESTED** [review]
Hey @dhruvilp! Great job so far with this!   I have just a couple comments.  I pushed a commit that properly ignores the `node_modules` folder.  It doesn't need to be included in the repo. 

- There seem to be some CSS issues. 
  - It seems that the HackRU logo isn't centered on larger screens.  screen shot 2019-01-14 at 3 48 38 pm
  - It also seems to look weird on mobile devices
  - On mobile devices it seems that some of the text is running off of the page.  This is particularly important for the packages sections.  Our finance team wants to be able to pull this up on their phones so we definitely need this to look good. 

Once those get resolved I think we'll be able to push live :) Let me know if anything else needs clarification.

File: frontend/public/index.html (hunk 1 above)
Comment (dhruvilp) [reply frontend/public/index.html:25]: I couldn't get the new favicon so I changed it to this one. Is there a way to update the old one?

File: frontend/src/MLHBadge.jsx (hunk 2 above)
Comment (dhruvilp) [reply frontend/src/MLHBadge.jsx]: no, I was trying to make it visible on all the sections of the page but not needed!

File: readme.md (hunk 3 above)
Comment (dhruvilp) [reply readme.md:53]: Well, I didn't make any changes to the Readme file and .gitignore file. I don't know who removed it!

**Comment by dhruvilp** [comment]
yeah, I'll fix those CSS issues.

File: frontend/src/MLHBadge.jsx (hunk 2 above)
Comment (thekabistro) [reply frontend/src/MLHBadge.jsx]: Ah.  I intentionally got rid of that lol.  If you want it back, make it `display: "absolute"`.

File: frontend/public/index.html (hunk 1 above)
Comment (dhruvilp) [reply frontend/public/index.html:25]: i fixed it!

**Comment by dhruvilp** [comment]
Good to go!!!

**Comment by thekabistro** [comment]
@dhruvilp 
screen shot 2019-01-16 at 9 06 35 am

Looks like a 1024 width results in this :\

**Comment by thekabistro** [comment]
We should also and get the testimonials in the page as actual text and not images, but we can do this later.

**Comment by dhruvilp** [comment]
> @dhruvilp
> screen shot 2019-01-16 at 9 06 35 am
> 
> Looks like a 1024 width results in this :\

Ok, I can fix this and also look into testimonials as well.

**Comment by thekabistro** [comment]
This looks really good!! There still seems to be a slight margin on the right side of the page whenever the mobile view is invoked.  We're almost ready to push!

File: frontend/src/components/Sponsorship/Sections/Recap.jsx [hunk 4]
```diff
@@ -1,17 +1,115 @@
...
+                    <div>
+                        <h4 class="zoom">88%</h4><div><h3>Undergraduates Hackers</h3></div>
                     </div>
                 </div>
-             </div>
+            </div>
+            <div class="col-lg-3">
+                <div class="hpanel testimo-panel text-center pd">
+                    <div>
+                        <h4 class="zoom">575 <i class="fa fa-caret-up gold fa-sm"></i></h4><div><h3>Totol Hackers</h3></div>
```
Comment (thekabistro) [inline frontend/src/components/Sponsorship/Sections/Recap.jsx]: Oops. Slight typo here.  Totolly missed it.

**Review by thekabistro — CHANGES_REQUESTED** [review]

**Review by thekabistro — APPROVED** [review]

**Pull request merged**